Collapsible.js

```diff
@@ -198,9 +198,7 @@ export default class Collapsible extends Component {
           style={[this.props.style, contentStyle]}
           onLayout={this.state.animating ? undefined : this._handleLayoutChange}
         >
-          <View style={{ height: measured ? contentHeight : null }}>
```

Comment:
ℹ️ `this._handleLayoutChange` will never be called if an explicit height is set here.

Reply:
In our project, we've got a calendar view that can change height depending on which month is selected. So what's happening in master is that `_handleLayoutChange` is called on expansion, measures it at 350 pixels, locks it to 350 pixels in this wrapping view, then when the user navigates to a month that spans 6 weeks like December, the height of the component changes to 370 or so -- but because of this wrapping view, `handleLayoutChange` is never called because the height was locked to 350 on initial measure.